packages/block-editor/src/components/block-navigation/tree.js

```diff
@@ -40,11 +45,13 @@ export default function BlockNavigationTree( {
 			__experimentalFeatures,
 			__experimentalPersistentListViewFeatures,
 			blockDropTarget,
+			isTreeGridMounted: isMounted.current,
 		} ),
 		[
 			__experimentalFeatures,
 			__experimentalPersistentListViewFeatures,
 			blockDropTarget,
+			isMounted.current,
```

Comment:
Just found this and wanted to chime in with a small feedback: Ref values (`ref.current`) shouldn't be included in the deps array as suggested by the [React team](https://twitter.com/dan_abramov/status/1105918794239803392) (and the [lint rule](https://github.com/WordPress/gutenberg/pull/24914)).